packages/playground-server/.env-cmdrc

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Can we add a note in the README regarding what this file is for / why we aren't using standard `.env`

Reply:
@snario Sure.